Signal/translations/ar.lproj/Localizable.strings

```diff
@@ -665,7 +665,7 @@
 "FINGERPRINT_SCAN_VERIFY_BUTTON" = "وضع علامة \"تم التحقق\"";
 
 /* No comment provided by engineer. */
-"FINGERPRINT_SHRED_KEYMATERIAL_BUTTON" = "إعادة تعيين هذه الجلسة.";
+"FINGERPRINT_SHRED_KEYMATERIAL_BUTTON" = "إعادة تعيين هذه الجلسة";
```

Comment:
"Reset Session" button label has a trailing period, which is wrong.  But maybe we should just leave it, rather than force re-translation in all of these languages.